docs/ref/contrib/admin/index.txt

```diff
@@ -2904,6 +2910,44 @@ The presence of the ``admin_password_reset`` named URL will cause a "forgotten
 your password?" link to appear on the default admin log-in page under the
 password box.
 
+.. _overriding-default-adminsite:
+
+Overriding the default ``AdminSite``
+------------------------------------
+
+.. versionadded:: 2.1
+
+You can override the default `django.contrib.admin.sites.AdminSite` instance with your own:
+set the `default_site` attribute to the dotted import path of your custom `AdminSite` subclass.
+
+.. snippet::
+    :filename: myproject/admin.py
+
+    class MyAdminSite(admin.AdminSite):
+        ...
+
+.. snippet::
+    :filename: myproject/apps.py
+
+    from django.contrib.admin.apps import AdminConfig
+
+    class MyAdminConfig(AdminConfig):
+        default_site = 'myproject.admin.MyAdminSite'
+
+.. snippet::
+    :filename: myproject/settings.py
+
+    INSTALLED_APPS = [
+        ...
+        #'django.contrib.admin.apps.AdminConfig',
+        'myproject.apps.MyAdminConfig',
+        ...
+    ]
+
+.. note::
+
+    Be sure that your `MyAdminConfig.label` has the default 'admin' value.
```

Comment:
This note confused me. 

Either: 

* This needs more explanation here. "`AdminConfig.label` is `admin` . — it is important we don't override that because..."
* Or, we can (redundantly) include it above with the the `MyAdminConfig` def. (Again maybe with a comment, `# This is the default but it's important...`)

Reply:
I think it is better to drop it, it adds nothing